database/consensus.go

```diff
@@ -336,7 +337,20 @@ WHERE date = $4`
 	return nil
 }
 
+func (db *Db) SetTxSender(tx *junotypes.Tx) error {
+	stmt := `UPDATE transaction SET sender = $1 WHERE hash LIKE $2`
+
+	hash := tx.TxHash[:len(tx.TxHash)-1] + "%"
```

Comment:
why not tx.Hash[:]

Reply:
Because hash is 64 symbols long and it isn't possible to search in database by 63+ symbols long string